docs/content/policy-reference.md

```diff
@@ -878,35 +878,36 @@ to automatic performance optimizations that are applied during policy evaluation
-| `raise_error` | no | `bool` | If `raise_error` is set, `http.send` will return an error that can halt policy evaluation when used in conjunction with the `strict-builtin-errors` option. Default: `true`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
-| `max_retry_attempts` | no | `number` | Number of times to retry a HTTP request when a network error is encountered. If provided, retries are performed with an exponential backoff delay. Default: `0`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `cache_ignored_headers` | no | `list` | List of header keys from `headers` parameter that should not considered when interacting with the cache. Default is `nil`, meaning all headers will be considered. **Important:** Note that if a cache entry exists with a subset/superset of headers that are considered in this request, it will lead to a cache miss.                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
> **Important:** Note that if a cache entry exists with a subset/superset of headers that are considered in this request, it will lead to a cache miss.

Can you provide an example of this please?

Reply:
Added an unit test for this [here](https://github.com/open-policy-agent/opa/pull/6675/files#diff-51c6c5bd2ad2a3fe3f9a303771fa5c82ae818d15ed0bae1560384c8b88672e92R1036).